docs/setup-cluster/k8s/setup-multiple-resource-managers.rst

```diff
@@ -0,0 +1,167 @@
+
+- Set up your additional clusters. These can be from the same or different clouds (e.g., GKE, AKS, EKS).
+- Gather the credentials for each cluster.
+
+For example:
+
+.. code:: bash
+
+   # for AKS az aks get-credentials --resource-group <resource-gp-name> --name <rm-name> # for
+   GKE gcloud container clusters get-credentials <rm-name>
+
+-  Using the cluster as the current context, save its ``kubeconfig`` to a ``tmp`` file.
+-  Repeat the above steps as many times as needed for the additional clusters you want to add.
+
+Next, switch to the cluster/context that you want to use as the default cluster. Then, repeat the
+following steps to create secrets for each additional cluster you want to add.
+
+-  Create a Kubernetes secret, from the ``tmp`` files for each additional cluster.
+-  Specify each additional resource manager, and its kubeconfig secret/location in
+   ``helm/charts/determined/values.yaml``.
+-  For example:
+
+.. code:: yaml
+
+   additional_resource_managers: 
+   - resource_manager:
+      type: kubernetes name: <rm-name> namespace: default 
+      # or whatever other namespace you want to use 
+      kubeconfig_secret_name: <The secret name, from ``kubectl describe secret <rm-name>``> 
+      kubeconfig_secret_value: <The data value, from ``kubectl describe secret <rm-name>``> 
+      ... and any other specs you may want to configure ... 
+      resource_pools: 
+         -  pool_name: <rm-pool>
+
+-  Once all of your resource managers are added to helm values file, install the Helm chart.
+
+Setting the master IP/Port for different resource managers
+==========================================================
+
+For resource managers where the master IP/Port is not reachable by the additional resource managers,
+you will need to update your Helm chart values/configuration to match the external IP of the default
+determined deployment. Once the cluster administrator has the master IP of the default Determined
+deployment, all that's necessary is to upgrade the Helm deployment with that value as the master IP
+for the additional clusters.
+
+*******
+ WebUI
+*******
+
+In the WebUI, the resource manager name is visible for each resource pool.
+
+To view resource managers:
+
+-  In the WebUI, navigate to the cluster view.
+-  Each resource pool card displays **Resource Manager Name**.
+
+This field helps identify whether a resource pool is managed locally or by another manager, tagged
+as “Remote” if defined in the :ref:`master-config-reference` file.
+
+Visibility and Access
+=====================
+
+**Resource Manager Name** is only visible to administrators or users with permissions to define
+multiple resource managers. Users can view all resource pools along with each resource pool's
+manager name to help distinguish between local and remote resource pools.
+
+Usage Example
+=============
+
+After configuring an additional resource pool named "test”, sign in to the cluster to see both the
```

Comment:
After configuring an additional resource manager named "additional-rm" with a resource pool called "test", ...

Reply:
Also add this:

Suppose you have a unnamed default resource manager (so it defaults to "default" name) with an unnamed default resource pool (so that defaults to "default" too).